composer.json

```
@@ -46,7 +46,7 @@
     },
     "conflict": {
         "doctrine/annotations": "<1.13.1 || >=3.0",
-        "doctrine/dbal": "<2.13.1 || ~3.0.0 || >=4.0",
+        "doctrine/dbal": "<2.13.1 || ~3.0.0",
```

```suggestion
        "doctrine/dbal": "<2.13.1 || ~3.0.0 || >=5.0",
```

We can keep a conflict with the next major release, it may prevent users from installing this bundle and getting errors with DBAL 5.